test/controllers/test-logout.js

```diff
@@ -52,4 +52,26 @@ describe('logout', function() {
       .expect('Set-Cookie', /refresh_token=;/)
       .end(done);
   });
+
+  describe('when Accept header is set to text/html', function() {
```

Comment:
Why, just because it starts with lowercase doesn't mean all letters have to be lowercase? The accept header is written with a capital `A` so it only makes sense in my opinion.